### ISSUE TITLE
About the baseline

### ISSUE THREAD

**Issue by TangZhengxu** [MungoMeng/Survival-XSurv#4]
Hello authors,

Thank you very much for open-sourcing this interesting work. I successfully reproduced your method and benefited greatly from it. Thanks again.

While reading your paper, I noticed that you reproduced the TMSS work. I would like to know if you used the 2022 HECKTOR dataset for this reproduction. I found that the EHR data in the 2022 HECKTOR is much less compared to the 2021 version. Did you base your reproduction of the TMSS work on the 2021 data, or did you use the 2022 dataset? If you used the 2022 dataset, could you please share which EHR data you selected as input for the clinical information?

Thank you very much!

**Comment by MungoMeng** [comment]
We used Hecktor2022 dataset in the XSurv study. We selected the clinical features using univariate and multivariate Cox analyses. Therefore, the choice is specific for different data splits. In our experiments (with our data split), the selected (best-performing)clinical features are Weight, HPV_1.0, Performance Status_1, and Performance Status_3. The categorical features have been preprocessed by get_dummies() function.
